samples/nrf9160/asset_tracker/README.rst

```diff
@@ -71,6 +72,10 @@ The sample is built as a non-secure firmware image, thus with ``CONFIG_TRUSTED_E
 It can be programmed independently from the secure boot firmware.
 
 See :ref:`gs_programming` for information about how to build and program the application.
+The Kconfig file of the sample contains options to configure the sample.
+For example, configure :option:`CONFIG_POWER_OPTIMIZATION_ENABLE` to enable power optimization or :option:`CONFIG_TEMP_USE_EXTERNAL` to use an external temperature sensor instead of simulated temperature data. 
```

Comment:
```suggestion
For example, configure ``CONFIG_POWER_OPTIMIZATION_ENABLE`` to enable power optimization or ``CONFIG_TEMP_USE_EXTERNAL`` to use an external temperature sensor instead of simulated temperature data.
```

Reply:
Sorry, messed up here - we can only link to config options for libraries etc., not for samples.